src/indexer/utils/evm/index.ts

```diff
@@ -253,6 +258,20 @@ export async function saveDepositLogs(
     }
   }
 
+  let senderStatus: string
+
+  try {
+    senderStatus = await ofacComplianceService.checkSanctionedAddress(sender)
+  } catch (e) {
+    logger.error((e as Error).message)
```

Comment:
Add a bit more context here (Checking sanctioned address failed...) and also check what error is returned when the api call fails (probably the response) and add that as context.